test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -40,27 +40,25 @@
 //
 // {
 //     {
-//         let _1 := 1
-//         let _2 := 15
-//         let _3 := 10
-//         let _4 := gt(not(gcd(_3, _2)), _1)
-//         let _5 := gcd(_3, _2)
-//         let _6 := not(0)
-//         let _7 := lt(or(_1, add(gcd(_3, _2), _6)), _1)
-//         let _8 := gcd(_3, _2)
-//         let _9 := gcd(_3, _2)
-//         pop(keccak256(gcd(_3, _2), or(gt(not(gcd(_3, _2)), _1), _1)))
-//         mstore(lt(or(gt(_1, or(or(gt(or(or(or(gt(or(gt(_6, _9), _1), _8), _7), _5), _1), _1), _4), _1)), _1), _1), _1)
-//         sstore(not(gcd(_3, _2)), _1)
-//         sstore(2, _1)
-//         extcodecopy(_1, msize(), _1, _1)
+//         let _1 := gt(not(gcd(10, 15)), 1)
+//         let _2 := gcd(10, 15)
+//         let _3 := lt(or(1, add(gcd(10, 15), not(0))), 1)
+//         let _4 := gcd(10, 15)
+//         let _5 := gcd(10, 15)
+//         pop(keccak256(gcd(10, 15), or(gt(not(gcd(10, 15)), 1), 1)))
+//         mstore(lt(or(gt(1, or(or(gt(or(or(or(gt(or(gt(not(0), _5), 1), _4), _3), _2), 1), 1), _1), 1)), 1), 1), 1)
+//         sstore(not(gcd(10, 15)), 1)
+//         sstore(2, 1)
+//         foo_singlereturn()
 //         sstore(0, 0)
-//         sstore(3, _1)
+//         sstore(3, 1)
 //     }
 //     function gcd(_a, _b) -> out
 //     {
 //         switch _b
 //         case 0 { out := _a }
 //         default { out := gcd(_b, mod(_a, _b)) }
 //     }
+//     function foo_singlereturn()
+//     { extcodecopy(1, msize(), 1, 1) }
```

Comment:
This looks odd. Is the test buggy? The name of `foo_singlereturn()` suggests it should return a single value and the function technically does have a single `out` return, but that's never assigned to and gets optimized out. Hard to tell if if was intentional or not.

In any case, the main thing that changes here is that the function does not get inlined. Note that it wasn't being inlined ether back in 0.5.11 when the test was added (0c0b5a09cd231c65a9c1890dcf9d2fd73f18d944), so not sure if I should classify it as a regression.